modules/API/pages/index.adoc

```diff
@@ -180,45 +181,47 @@ TYPEDEF TUPLE <field1 INT(1), field2 UINT, field3 STRING(10), field4 DOUBLE> myT
 ----
 
 
-==== Vertices with Composite Keys
+==== Vertices with composite keys
```

Comment:
Please also add such case into "Complex type parameter": https://docs.tigergraph.com/gsql-ref/current/querying/query-operations#_parameter_list_2

Create Query:
CREATE QUERY q_v_person(VERTEX<compositePerson> v) { PRINT v; }
CREATE QUERY q_v_generic(VERTEX v) { PRINT v; }

GSQL Run query:
GSQL > RUN QUERY q_v_person( ("Bob", 123) )
GSQL > RUN QUERY q_v_generic( (("Tom", 456), "compositePerson") )

Reply:
yes, i can add this; that is a different repo though and I'll do a separate PR